vscode-extension/src/extension.ts

```diff
@@ -147,6 +158,33 @@ export function activate(context: vscode.ExtensionContext) {
       });
     })
   );
+
+  const pythonapi = await PythonExtension.api();
+  context.subscriptions.push(
+    pythonapi.environments.onDidChangeActiveEnvironmentPath(
+      async (e: ActiveEnvironmentPathChangeEvent) => {
+        await savePythonInterpreterToCache();
+        const editors: Array<AIConfigEditorState> = Array.from(
+          aiconfigEditorManager.getRegisteredEditors()
+        );
+        if (editors.length > 0) {
+          vscode.window
+            .showInformationMessage(
+              "Python Interpreter Updated: Would you like to refresh active AIConfig files?",
```

Comment:
nit

```suggestion
              "Python Interpreter Updated: Would you like to restart active AIConfig file servers?",
```

Reply:
Synced offline w. Sarmad "file servers" is a bit confusing for regular people to understand, but we should have a learn more button so people can understand why on interpreter update we need to refresh